src/SearchForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+export default function SearchForm({initialQuery,onSearch}) {
+
+    const [query,SetQuery] = useState(initialQuery);
```

Comment:
You can try using the defaultValue property to avoid this state if the requirement is to access the value only on form submission